access_identity_provider.go

```diff
@@ -43,6 +43,7 @@ type AccessIdentityProviderConfiguration struct {
 	SsoTargetURL       string   `json:"sso_target_url,omitempty"`
 	SupportGroups      bool     `json:"support_groups,omitempty"`
 	TokenURL           string   `json:"token_url,omitempty"`
+	PKCEEnabled        bool     `json:"pkce_enabled,omitempty"`
```

Comment:
is this value always present in the request/responses? do we need a `*bool` here?

Reply:
Nope, but neither are any of the other fields. The server only returns values that are configured. The same applies for pkce_enabled=false

Do you think this will cause issues?

Reply:
as long as "not present" in the payload is an ok state and we default accordingly. we have other APIs at the moment that need `true`/`false`/`uninitialised` and it does cause an issue in Terraform due to the inability to set null values for booleans.

Reply:
Oh, we should be fine then. There is no need to distinguish false from uninitialised